common/src/components/accordion/Section.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import clsx from "clsx";
+import { ChevronDownIcon, ChevronRightIcon } from "@heroicons/react/solid";
+
+export interface ExternalSectionProps {
+  title: string;
+  subtitle?: string;
+  Icon?: React.ComponentType<React.SVGProps<SVGSVGElement>>;
```

Comment:
hmm .. lower casing had issues with the custom tag like <icon> was not acceptable but <Icon> does.

Reply:
will try again anyways

Reply:
Found the reason.. because its a component type. It has to start with capitals 
https://stackoverflow.com/questions/37414304/typescript-complains-property-does-not-exist-on-type-jsx-intrinsicelements-whe